documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -103,8 +103,8 @@ spec:
 <2> Name to identify the listener. Must be unique within the Kafka cluster.
 <3> Port number used by the listener inside Kafka. The port number has to be unique within a given Kafka cluster. Allowed port numbers are 9092 and higher with the exception of ports 9404 and 9999, which are already used for Prometheus and JMX. Depending on the listener type, the port number might not be the same as the port number that connects Kafka clients.
 <4> External listener type specified as `route`, `loadbalancer`, `nodeport` or `ingress`. An internal listener is specified as `internal` or `cluster-ip`.
-<5> Enables TLS encryption on the listener. Default is `false`. TLS encryption is not required for `route` listeners.
-<6> Authentication specified as mutual `tls`.
+<5> TLS encryption on the listener. Default is `false`. For mTLS authentication, the configuration also enables server authentication. TLS encryption is not required for `route` listeners.  
```

Comment:
```suggestion
<5> Required. TLS encryption on the listener. For `route` and `ingress` type listeners it has to be set to `true`. To mTLS authentication,  use the `authentication` section.
```

Reply:
Changed to:

> Required. TLS encryption on the listener. For `route` and `ingress` type listeners it must be set to `true`. For mTLS authentication, also use the `authentication` property. 

I've also updated elsewhere in the doc related to route and ingress config and `tls: true`